flex-box-template-style.css

```diff
@@ -0,0 +1,117 @@
+.layout{
+    display:flex;
+    justify-content: center;
+    margin: 0 auto;
+    width: 100%;
+    height: 100%;
+    flex-direction: column;
+
+
+}
+.container{
+    display:flex;
+    width: 700px;
+    height:auto;
+}
```

Comment:
you should leave a space after each CSS blog.  
for example;
.container {
.
.
.
.
}

.nav{
.
.
.
}